repose-aggregator/core/repose-core/src/main/java/org/openrepose/core/services/datastore/distributed/impl/ehcache/ReposeLocalCache.java

```diff
@@ -35,8 +34,7 @@
 import java.security.NoSuchAlgorithmException;
 
 @Named("reposeLocalDatastore")
-@ManagedResource(objectName = MBeanObjectNames.REPOSE_LOCAL_CACHE,
-        description = "Repose local datastore MBean.")
+@ManagedResource(description = "Repose local datastore MBean.")
```

Comment:
No longer need the `objectName` since, again, it was just repeating the package and classname.